
This patch:
-fixes non-working Eth on Rock-3a v1.31
-corrects CEC to use hdmitxm1_cec1 as rock-3a has HW change switchnig CEC from hdmitxm1_cec0 to hdmitxm1_cec1

diff --speed-large-files --no-dereference --minimal -Naur linux-6.1-rc2/arch/arm64/boot/dts/rockchip/rk3568-rock-3a.dts linux-6.1-rc2/arch/arm64/boot/dts/rockchip/rk3568-rock-3a.dts
--- linux-6.1-rc2/arch/arm64/boot/dts/rockchip/rk3568-rock-3a.dts	2022-10-24 00:27:33.000000000 +0200
+++ linux-6.1-rc2/arch/arm64/boot/dts/rockchip/rk3568-rock-3a.dts	2022-10-26 10:24:19.706624874 +0200
@@ -211,7 +219,14 @@
 	assigned-clock-rates = <0>, <125000000>;
 	clock_in_out = "output";
 	phy-handle = <&rgmii_phy1>;
-	phy-mode = "rgmii-id";
+	phy-mode = "rgmii";
+	tx_delay = <0x42>;
+	rx_delay = <0x28>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&eth_phy_rst>;
+	snps,reset-gpio = <&gpio3 RK_PB0 GPIO_ACTIVE_LOW>;
+	snps,reset-active-low;
+	snps,reset-delays-us = <0 20000 100000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&gmac1m1_miim
 		     &gmac1m1_tx_bus2
@@ -229,6 +244,9 @@
 &hdmi {
 	avdd-0v9-supply = <&vdda0v9_image>;
 	avdd-1v8-supply = <&vcca1v8_image>;
+	assigned-clocks = <&cru CLK_HDMI_CEC>;
+	assigned-clock-rates = <32768>;
+	pinctrl-0 = <&hdmitx_scl &hdmitx_sda &hdmitxm1_cec>;
 	status = "okay";
 };
 
@@ -530,11 +548,6 @@
 	rgmii_phy1: ethernet-phy@0 {
 		compatible = "ethernet-phy-ieee802.3-c22";
 		reg = <0x0>;
-		pinctrl-names = "default";
-		pinctrl-0 = <&eth_phy_rst>;
-		reset-assert-us = <20000>;
-		reset-deassert-us = <100000>;
-		reset-gpios = <&gpio3 RK_PB0 GPIO_ACTIVE_LOW>;
 	};
 };
 
